cutter/cutter/main.py

```diff
@@ -315,11 +327,14 @@ def upload_wrapper():
 			)
 		except JobConsistencyError:
 			raise # this ensures it's not caught in the next except block
+			upload_errors.labels(video_channel=job.video_channel, video_quality=job.video_quality, upload_location=job.upload_location).inc()
 		except ErrorHandled:
 			# we're aborting the cut, error handling has already happened
+			upload_errors.labels(video_channel=job.video_channel, video_quality=job.video_quality, upload_location=job.upload_location).inc()
 			return
 		except Exception as ex:
 			self.refresh_conn()
+			upload_errors.labels(video_channel=job.video_channel, video_quality=job.video_quality, upload_location=job.upload_location).inc()
```

Comment:
we should distinguish between errors in the cut vs errors in the upload, because cut errors are considered unretriable (final state is unedited) but upload errors are (final state is edited), under the assumption it's a transient network issue (though it could also be a youtube api issue which will fail every time).

though, we'd also want to distinguish between upload errors before / after finalizing, since ones after arent retriable and require operator intervention.

new plan: we could have one metric `upload_errors`, with a label for final state, ie. `unedited`, `edited` or `finalizing`.